korangar_audio/src/settings.rs

```diff
@@ -0,0 +1,52 @@
+#[cfg(feature = "debug")]

```

Comment:
Please don't move the settings into the audio crate. It should be agnostic of application specific code.

The graphic engine currently only has the "GraphicsSettings", because we haven't move it to it's own crate yet.

Reply:
I am going to move the files to korangar/audio/settings.